apps/k8s-io/configmap-nginx.yaml

```
@@ -54,6 +54,11 @@ data:
             ';
           }
 
+          # Redirect go import paths to Go package documentation.
+          if ($repo ~ "api|apimachinery|client-go|kubernetes") {
+            return 301 https://pkg.go.dev/k8s.io/$repo/$subpath;
```

I'm still not familiar with this config but why not :
```suggestion
            return 301 https://git.k8s.io/$repo/$subpath;
```
?
I think you can get the same result.

---

This would end up redirecting to eg github.com/kubernetes/client-go, and I'm trying to get this to redirect to Go package docs at [pkg.go.dev/k8s.io/client-go](https://pkg.go.dev/k8s.io/client-go)